esbuild/hooks.sh

```diff
@@ -1,3 +1,4 @@
 esbuild src/hooks/index.ts --outfile=hooks/index.js \
```

Comment:
In last version on mermaid-extension, I replace hooks folder with react.
Can you make the same thing?

Reply:
Done